en/docs/adv_examples/bert.md

```diff
@@ -115,96 +116,84 @@ average speed: 0.556(sentences/sec)
 ------------------------------------------------------------------
 ```
 
-## 详细说明
-### 脚本说明
-|            **分类**            |                                **说明**                                 |   **所属**   |
-|:----------------------------:|:---------------------------------------------------------------------:|:----------:|
-|     pretrain.py、bert.py      |                             定义了BERT网络模型；                              |    BERT    |
-|      run_pretraining.py      | 启动BERT训练的用户脚本，用户通过命令行参数进行BERT训练的训练环境及超参配置，各个参数的具体作用将在下文 **脚本参数** 中说明。 |    BERT    |
-|           squad.py           |                              定义了squad网络；                              |   SQuAD    |
-|         run_squad.py         |                             用于启动SQuAD的训练                              |   SQuAD    |
-|    run_squad_predict.py    |                           使用训练好的SQuAD模型进行预测                           |   SQuAD    |
-|         npy2json.py          |                将OneFlow的预测结果转化为prediction json格式的必要脚本                 |   SQuAD    |
-| convert_tf_ckpt_to_of.py |                      将TensorFlow模型转为OneFlow个模型格式                      | BERT/SQuAD |
-
-
-
-### 脚本参数
-`run_pretraining.py`通过命令行参数配置包括超参在内的训练环境，可以通过 `run_pretraining.py --help`查看，以下是这些参数作用的具体说明：
+## Detailed description
+### Scripts
+| **Files** | **Description** | **Belongs to**|
+|:---------:|:----------:|:----------:|
+|pretrain.py、bert.py| Define the BERT model |BERT|
+|run_pretraining.py|Start BERT training. The user can configure the training environment and parameters of the bet training through the command line parameters. The specific meanings of each option will be described in the **script options** below.| BERT|
```

Comment:
|run_pretraining.py|Start BERT training. The user can configure the training environment and parameters of the bert training through the command line parameters. The specific meanings of each option will be described in the **script options** below.| BERT|